readme.md

```diff
@@ -34,6 +34,8 @@
 ### NMEA0183 Libraries
 - [Python](https://github.com/Knio/pynmea2)
 
+### GPIO Libraries
```

Comment:
Let's keep to only marine/boating specific resources or else this will expand infinitely 🙂

Reply:
Hat Labs Shop provides Open Marine Electronics PCB and WiringPi can be used for sensors (temperature, humidity, atmospheric pressure etc..)

Reply:
Yes, supportive of HatLabs, but not WiringPi as it's a generic GPIO library and not in any way boating specific.  While it may be one of the top ones, there are [many more GPIO RPi libraries](https://docs.google.com/spreadsheets/d/1sFCJuPZ9k5GN0A6j7gHRckvbwIdnwzPH1sTCuPNFNRQ/edit?gid=0#gid=0) that are equally not boating specific.  While I realize it might be helpful to some to have other, non-boating, libraries/tools listed here, it will ultimately lead to 1) diluting the list with non-boating content and 2) making opinionated choices about which non-boating libraries get included. 

I hope this makes sense.  Thanks!